app/src/main/java/com/eveningoutpost/dexdrip/BluetoothScan.java

```diff
@@ -309,10 +309,15 @@ protected void onListItemClick(ListView l, View v, int position, long id) {
             if (!
                     (CollectionServiceStarter.isBTWixel(getApplicationContext())
                             || CollectionServiceStarter.isWifiandBTWixel(getApplicationContext())
-                    )) {
+                    ) || CollectionServiceStarter.isLimitter(getApplicationContext())) {
```

Comment:
If we have indeed seen drip in the name than I think we should change even if the user has chosen Limitter. 
This is probably a user mistake.